app/src/main/java/org/p2p/wallet/rpc/repository/amount/RpcAmountInteractor.kt

```diff
@@ -0,0 +1,26 @@
+package org.p2p.wallet.rpc.repository.amount
+
+import org.koin.ext.getFullName
+import timber.log.Timber
+import java.math.BigInteger
+
+class RpcAmountInteractor(
```

Comment:
I think this interactor should have at least 3 functions:
`getFees`
`getLamportsPerSignature`
`getMinBalanceForRentExemption`.

Please look into `TransactionAmountInteractor` and leave one of two interactors